artifactory/commands/buildinfo/publish.go

```diff
@@ -91,12 +94,51 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	artVersion, err := servicesManager.GetVersion()
+	if err != nil {
+		return err
+	}
+	artVersionToStr, err := strconv.Atoi(artVersion[0:1])
+	if err != nil {
+		return err
+	}
+	log.Info(bpc.printBuildInfoLink(artVersionToStr, buildTime))
+
 	if !bpc.config.DryRun {
 		return utils.RemoveBuildDir(bpc.buildConfiguration.BuildName, bpc.buildConfiguration.BuildNumber, bpc.buildConfiguration.Project)
 	}
 	return nil
 }
 
+func (bpc *BuildPublishCommand) printBuildInfoLink(artVersion int, buildTime time.Time) string {
+	if artVersion <= 6 {
+		return "Build info successfully deployed. Browse it in Artifactory under " +
+			bpc.serverDetails.GetUrl() +
+			"artifactory/webapp/#/builds/" +
+			bpc.buildConfiguration.BuildName + "/" + bpc.buildConfiguration.BuildNumber
+	} else if bpc.buildConfiguration.Project != "" {
+		timestamp := buildTime.UnixNano() / 1000000
+		return "Build info successfully deployed. Browse it in Artifactory under " +
+			bpc.serverDetails.GetUrl() +
+			"ui/builds/" +
+			bpc.buildConfiguration.BuildName + "/" + bpc.buildConfiguration.BuildNumber + "/" + strconv.FormatInt(timestamp, 10) +
+			"/published" + "?buildRepo=" + bpc.buildConfiguration.Project + "-build-info&projectKey=" + bpc.buildConfiguration.Project
+	} else {
+		timestamp := buildTime.UnixNano() / 1000000
+		return "Build info successfully deployed. Browse it in Artifactory under " +
+			bpc.serverDetails.GetUrl() +
+			"ui/builds/" +
+			bpc.buildConfiguration.BuildName + "/" + bpc.buildConfiguration.BuildNumber + "/" + strconv.FormatInt(timestamp, 10) +
+			"/published?buildRepo=" + bpc.buildConfiguration.Project + "artifactory-build-info"
+	}
+}
```

Comment:
Let's use ```fmt.Sprintf``` to build all the above URLs. It looks better, and also more readable.